<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>160</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QSpinBox" name="spinBox_8"/>
     </item>
     <item row="8" column="0">
      <widget class="QSpinBox" name="spinBox_9"/>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>K</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QSpinBox" name="spinBox_4"/>
     </item>
     <item row="9" column="0">
      <widget class="QSpinBox" name="spinBox_10"/>
     </item>
     <item row="5" column="0">
      <widget class="QSpinBox" name="spinBox_6"/>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>E</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QSpinBox" name="spinBox"/>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>J</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QSpinBox" name="spinBox_5"/>
     </item>
     <item row="6" column="0">
      <widget class="QSpinBox" name="spinBox_7"/>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>G</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QSpinBox" name="spinBox_11"/>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>H</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QSpinBox" name="spinBox_3"/>
     </item>
     <item row="1" column="0">
      <widget class="QSpinBox" name="spinBox_2"/>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>D</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>F</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>I</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>10</y>
      <width>181</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Последовательность:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Медианные значения:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>353</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
